ST_structs.h

```diff
@@ -1,333 +1,721 @@
-                       * EG start year = 96 I will have 5 flags for one annual species
-                       * for year 96,97,98,99,100. BUT extirpate will not un-flag this*/
+      /** \brief individuals in group killed. Index by age at death. */
+  IntUS *kills,
+      /** \brief Total indivs in group established during current iteration. */
+        estabs,
+      /** \brief If killyr equals the current year there will be a fire.
+       * \sa killfreq
+       * \sa mort_EndOfYear() */
+        killyr,
+      /** \brief Accumulator for consecutive years of low resources. */
+        yrs_neg_pr,
+      /** \brief Treated as a Bool. If TRUE there was a wildfire in the current year. 
+       * \sa mort_EndOfYear() */
+        wildfire,
+      /** \brief Treated as a Bool. If TRUE there was a prescribed fire in the current year. 
+       * \sa mort_EndOfYear() */
+        prescribedfire,
+      /** \brief Xxtra resources in millimeters */
+        mm_extra_res;
+      /** \brief Resources required for current biomass of resgroup. */
+  RealF res_required,
+      /** \brief Resources available adjusted to account for competition. */
+        res_avail,
+      /** \brief Resource applied to superficial growth. */
+        res_extra,
+      /** \brief Ratio of resources required to resources available */
+        pr,
+      /** \brief Fraction of the total biomass for this vegtype contributed by this resgroup.
+       * \sa veg_prod_type */
+        rgroupFractionOfVegTypeBiomass;
+      /** \brief Number of species established in group. */
+  SppIndex est_count,
+      /** \brief Array of indexes of species established in this resgroup. 
+       * \sa Species */
+           *est_spp;
+      /** \brief If TRUE this group is extirpated, no more regeneration. */
+  Bool extirpated,
+      /** \brief For annuals: If TRUE this group can regenerate. */
+       regen_ok;
 
   /**** Quantities that DO NOT change during model runs *****/
 
-  IntUS max_stretch,    /* num yrs resources can be stretched w/o killing*/
-        max_spp_estab,  /* max # species that can add new plants per year*/
-        max_spp,        /* number of species in the group*/
-        max_age,        /* longest lifespan in group. used to malloc kills[] */
-        startyr,        /* don't start trying to grow until this year */
-	killfreq_startyr,/* start year for kill frequency*/
-
-        extirp,         /* year in which group is extirpated (0==ignore) */
-        grp_num,        /* index number of this group */
-        veg_prod_type, /* type of SOILWAT2 vegetation type:
-          STEPWAT2 inputs via "rgroup.in" are defined as: 1 for tree, 2 for shrub, 3 for grass, 4 for forb;
-          however, the inputs get translated by get_SW2_veg_index() to SOILWAT2 values immediately upon reading the inputs (see SW_Defines.h) */
-		grazingfrq,     /* grazing effect on group at this frequency: <1=prob, >1=# years */
```

Comment:
this is not longer present below! was accidentally deleted

Reply:
`grazingfrq` has been moved from the INTUS declarations to the RealF declarations. This should have already happened, but `grazingfrq` needs to be a RealF to allow for values between 0 and 1 (stochastic grazing).